packages/waku/src/lib/types.ts

```diff
@@ -8,9 +8,21 @@ type RenderRsc = (elements: Record<string, unknown>) => Promise<ReadableStream>;
 type RenderHtml = (
   elements: Elements,
   html: ReactNode,
-  options: { rscPath: string; actionResult?: unknown; status?: number },
+  options: { rscPath: string; actionResult?: any; status?: number },
 ) => Promise<Response>;
 
+type RenderHtmlWithStream = (
+  elementsStream: ReadableStream,
+  html: ReactNode,
+  options: { rscPath: string; actionResult?: any; status?: number },
+) => Promise<Response>;
+
+export type RenderUtils = {
+  renderRsc: RenderRsc;
+  renderHtml: RenderHtml;
+  renderHtmlWithStream: RenderHtmlWithStream;
```

Comment:
I don't love having both of them. Just change the signature of `renderHtml`.

Reply:
That would require all current usage of `renderHtml(elements< ...)` to `renderHtml(await renderRsc(elements), ...)`, but is that the breaking you mean?